aequilibrae/parameters.py

```diff
@@ -31,21 +33,29 @@ class Parameters:
 
     def __init__(self):
         """ Loads parameters from file. The place is always the same. The root of the package"""
-        path = os.path.dirname(os.path.realpath(__file__))
-        self.file = os.path.join(path, "parameters.yml")
-        with open(self.file, "r") as yml:
-            self.parameters = yaml.load(yml, Loader=yaml.SafeLoader)
+
+        proj_path = os.environ.get('AEQUILIBRAE_PROJECT_PATH', '')
```

Comment:
it's nice having an option to load from a path in an env var, but isn't there any other way to pass that information into this class... like arguments on the init method?

Reply:
The problem is that this is imported in different places by different procedures, and they would all need to get from the project that is being used.

This is also transient (it does not add anything permanently to the host system)